src/actions/firebase/firebase.ts

```diff
@@ -96,12 +96,13 @@ export class FirebaseAction extends Hub.Action {
             if (device.device_id && device.user_id) {
               const deviceId = device.device_id.toString()
               notificationData.user_id = device.user_id.toString()
-              const payload = {
+              const message = {
+                token: deviceId,
                 notification,
                 data: notificationData,
               }
               try {
-                await this.sendMessageToDevice(deviceId, webhookId, payload, notificationOptions)
```

Comment:
Not blocking, but does `notificationOptions` even need to exist if we're not passing it anywhere?  Looks like we're using it above to modify `this.notificationOptions`, but seems like you could just get rid of the const and directly modify `this.notificationOptions`?